### PR TITLE
[RHDX-128]First css refactor moving code around and removing from rhpd2.css

### PR DESCRIPTION
**Pull request by alistairmc**
### JIRA Issue Link
https://projects.engineering.redhat.com/browse/RHDX-128

### Verification Process
Review site and check there are no new css display issues.

I have created a follow-up task to move any additional styles https://projects.engineering.redhat.com/browse/RHDX-144

### CODE REVIEW

**Comment by alistairmc** [comment]
rebuild this please

**Comment by robpblake** [comment]
rebuild this please

**Comment by staceymosier** [comment]
@alistairmc   Please check the page https://developer-preview-3455.ext.us-west.dc.preprod.paas.redhat.com/courses/service-mesh/,  it no longer has a white background.

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/pages/_katacoda.scss [hunk 1]
```diff
@@ -0,0 +1,121 @@
+/* Katacoda */
+.katacoda-courses {
```
Comment (staceymosier) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/pages/_katacoda.scss]: There is a change in background on this page where .katacoda-courses is in used: https://developer-preview-3455.ext.us-west.dc.preprod.paas.redhat.com/courses/service-mesh/

Comment (alistairmc) [reply _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/pages/_katacoda.scss]: @staceymosier I have pushed a fix for the mentioned issue, please review.

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_assembly-base.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.assembly {
+    @extend %assembly;
+    /* Assembly Variations */
+    &[data-variant="dark"] {
+        --pfe-theme--color--surface--base: rgb(19, 20, 20);
+        --pfe-theme--font-size--alpha: 42px;
+        --pfe-theme--color--text: $white;
```
Comment (staceymosier) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_assembly-base.scss]: I'm investigating this in my current local branch.

**Pull request merged**